group/assignment5/GraphTest.java

```diff
@@ -0,0 +1,92 @@
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class GraphTest {
+
+  @Test
```

Comment:
you have 3 different tests about the DAG - maybe you could add more details in the method name so it is possible to catch the difference by only reading the title? (just like you did in the tests for the Alphabet class!)